packages/backend/src/models/SpaceModel.ts

```diff
@@ -358,6 +359,7 @@ export class SpaceModel {
             spaceUuid?: string;
             spaceUuids?: string[];
             slug?: string;
+            parentSpaceUuid?: string;
```

Comment:
we may need to change this to `string | null` after the rebase (if my changes go in first)